src/main/scala/dpla/ingestion3/enrichments/TypeEnrichment.scala

```diff
@@ -15,7 +15,6 @@ class TypeEnrichment extends VocabEnrichment[String] {
   val dcmiType = DCMIType()
 
   val DcmiTypeMap: Map[String, IRI] = Map(
-    "appliance" -> dcmiType.Image,
```

Comment:
Umm, what about my collection of washer/dryer photos?

Reply:
@mdellabitta and the dad jokes strike again :)

Reply:
Damn straight!

Reply:
laugh but I spent more time with this book in my public history class than I care to remember 
`https://www.amazon.com/Save-Womens-Lives-History-Machines/dp/0972971009`